installer/build-mac-dmg.sh

```diff
@@ -52,8 +53,11 @@ if [ -d "$OS_PATH/MacOS/python3.6" ]; then
   mv "$OS_PATH/MacOS/python36.zip" "$OS_PATH/Resources/python36.zip"; ln -s "../../Resources/python36.zip" "$OS_PATH/MacOS/lib/python36.zip";
 fi
 
+echo "Loop through bundled files and sign all binary files"
+find "build" \( -iname '*.dylib' -o -iname '*.so' \) -exec codesign -s "OpenShot Studios, LLC" --timestamp=http://timestamp.apple.com/ts01 --entitlements "installer/openshot.entitlements" --force "{}" \;
+
 echo "Code Sign App Bundle (deep)"
-codesign -s "OpenShot Studios, LLC" "build/$OS_APP_NAME" --deep --force
+codesign -s "OpenShot Studios, LLC" --force --deep --entitlements "installer/openshot.entitlements" --options runtime --timestamp=http://timestamp.apple.com/ts01 "build/$OS_APP_NAME"
```

Comment:
Every single file must be signed. If I don't sign with --deep, I get errors on *.svg files and all sorts of other files. Also, this adds a hardened runtime, a signature with a timestamp (required for notarization), and some entitlements, required to allow our @rpath handling.